docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        "Surv(time,status) ~ 1 + B(trt_str, success='Test') + T(celltype_str, ref='Squamous')",
</pre><p>I really like you're using <code>B()</code> and <code>T()</code> in the model formula. Again, one sentence saying what they do would be great. I could help here if you want since I was the author of these shortcuts.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>